pkg/scheduler/api/devices/nvidia/vgpu/device_info.go

```diff
@@ -114,6 +117,15 @@ func NewGPUDevices(name string, node *v1.Node) *GPUDevices {
 	return nodedevices
 }
 
+func (gs *GPUDevices) ScoreNode(pod *v1.Pod, schedulePolicy string) float64 {
+	/* TODO: we need a base score to be campatable with preemption, it means a node without evicting a task has
+	a higher score than those needs to evict a task */
+
+	// Use cached stored in filter state in order to avoid recalculating.
+	klog.V(3).Infof("Scoring pod %s with to node %s with score %f", pod.Name, gs.Name, gs.Score)
```

Comment:
`ssn.AddNodeOrderFn` has already print the score with log level 5, it's not necessary to print here cause it will print logs of logs.